src/gui/mzroll/mainwindow.cpp

```diff
@@ -2725,28 +2740,13 @@ void MainWindow::createMenus() {
 	QAction* faq = helpMenu->addAction("FAQs");
 	connect(faq, SIGNAL(triggered()), signalMapper, SLOT(map()));
 
-	QAction* start = helpMenu->addAction("Getting Started");
-	connect(start,SIGNAL(triggered(bool)), gettingstarted, SLOT(show()));
-
 	signalMapper->setMapping(doc, 1);
 	signalMapper->setMapping(tutorial, 2);
 	signalMapper->setMapping(faq, 3);
 
-	connect(signalMapper, SIGNAL(mapped(int)), this, SLOT(openURL(int)));
```

Comment:
The Help menu URLs are dependent on this. Has this been moved somewhere else?

Reply:
I just searched to make sure that this is not being used anywhere. Maybe you are confusing with `openUrl` which is a Qt utility for opening URLs. This removed one seems redundant.

Reply:
The `signalMapper` maps which link is opened on clicking which option in the Help menu. The `openURL` method uses the Qt method `openUrl` internally to open such links but I just checked and that method has been removed in this PR for some reason.

Just compiled it to confirm and the clicks aren't working.

Reply:
This should not have been removed for any reason. I will replace the functionality.